Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeView.cs

```diff
@@ -175,7 +183,7 @@ private void ItemsVectorChanged(IObservableVector<object> sender, IVectorChanged
         {
             if (BladeMode == BladeMode.Fullscreen)
             {
-                var bladeItem = (BladeItem)sender[(int)e.Index];
+                var bladeItem = (BladeItem)ContainerFromItem(sender[(int)e.Index]);
```

Comment:
why this is not GetBladeItem ?
Plus here ContainerFromitem is used with casting which means we're sure that item will always be BladeItem why we're using as in method GetBladeItem ?